src/component/axisPointer/axisTrigger.js

```diff
@@ -97,19 +97,92 @@ export default function (payload, ecModel, api) {
 
     // Process for triggered axes.
     each(coordSysAxesInfo.coordSysMap, function (coordSys, coordSysKey) {
+        /**
```

Comment:
I think this should be the default behavior. The previous design where axes have separate snaps is a bug rather than a feature. 
Take the [example](https://echarts.apache.org/examples/en/view.html?c=scatter-nutrients-matrix&edit=1&reset=1) of what you mentioned, Why is it even necessary to highlight the points that are far away from the mouse point while having the same x or y value? I don't think this should be the expected interaction. Instead, what I want to make is like [this highcharts demo](https://jsfiddle.net/ovilia/9tyhnegx/), which I think makes more sense.

Reply:
> Why is it even necessary to highlight the points that are far away from the mouse point while having the same x or y value? 

I think we can get more than one points highlighted on xAxisPointer or yAxisPointer. 
But if one axisPointer follow another, we can't. 
In fact, in that [example](https://echarts.apache.org/examples/en/view.html?c=scatter-nutrients-matrix&edit=1&reset=1) xAxisPointer and yAxisPointer are at the same level. No one is primary or secondary.

![image](https://user-images.githubusercontent.com/1956569/69695803-91ba1480-1118-11ea-925a-247174fa718d.png)

Reply:
I think `highlight` the nearest point is a more intuitive, and more common scenario. Yes sure the scene in the example @100pah  metioned also exists. But it's not that common. At least it's not as common as the highcharts demos. So I think a breaking change worth it. Also we can provide another option if developers still want the previous behaviour.

Reply:
>  I believe it worth a breaking change to change the default behaviour. Also we can provide another option if developers still want the previous behaviour.
I agree.

Reply:
@pissang Any progress on this? Thanks!